scripts/release.sh

```diff
@@ -0,0 +1,252 @@
+        sudo gem install jazzy
+    fi
+
+    # this runs Jazzy to generate the documentation
+    echo "Running Jazzy to generate documentation..."
+    jazzy --clean --objc --framework-root SmartDeviceLink --sdk iphonesimulator --umbrella-header SmartDeviceLink/public/SmartDeviceLink.h --theme theme --output docs
+fi
+
+# 7 Ensure that the RPC_SPEC has released to the master branch and update the submodule to point to the new release tag (or to the HEAD of master, if no release of the RPC_SPEC is occurring).
+echo 
+echo "The rpc spec site will be opened for you"
+echo "https://github.com/smartdevicelink/rpc_spec"
+echo "Please check if there is a new release of the RPC_SPEC to master."
+echo "If there is, please update the rpc_spec submodule to point to the new master, then press enter."
+read user_input
+open "https://github.com/smartdevicelink/rpc_spec"
+read user_input
+#TODO - phase ? - can this be automated.  Check version.  Check version at site.
+
+echo
+echo "Please perform the following steps to push the release to master:"
+# and every step of this process, the user needs complete control.
+# that means we tell them what we want to do, ask them if they want us to do it, report the results, and gracefully handle exits.
+
+# git commands
+# 8 commit release to develop
+echo "Would you like to commit these changes to the develop branch? Y/n"
+read user_input
+#if [[ ! $user_input == [Nn] ]]; then
+#todo - phase 3 - run the commands automatically
+echo "Please Commit and push the changes to develop"
+
+# 8 merge release to master
+echo "Would you like to merge this release to master? (This will not push to master) Y/n"
+read user_input
+#if [[ ! $user_input == [Nn] ]]; then
+#todo - phase 3 - run the commands automatically
+echo "Please push to master, then press enter"
+read user_input
+
+# 8 tag it
+echo "Would you like to tag this release? (This will not push the tag) Y/n"
+read user_input
+#if [[ ! $user_input == [Nn] ]]; then
+#todo - phase 3 - run the commands automatically
+
+# 8 merge master back to develop
+echo "Would you like to merge master back into develop? (This will not push the branch) Y/n"
+# git merge develop <master>
+read user_input
+#if [[ ! $user_input == [Nn] ]]; then
+#todo - phase 3 - run the commands automatically
+
+echo
+echo "Please perform the following steps"
+# 1 Create new release for tag
+echo "Would you like to go to the Github releases page to create a release? Y/n"
+read user_input
+if [[ ! $user_input == [Nn] ]]; then  
+    open "https://github.com/smartdevicelink/sdl_ios/releases"
+
+    # TODO - phase 3 - this can be automated with gh
+    # https://cli.github.com/manual/gh_release_create
+    # TODO - can we pull the list of changes from Changelog.md and automatically add those to the release (so we do not type the same things twice)
+    # TODO - if/when we automate this, make sure to open the releases page so the user can review it.
```

Comment:
```suggestion
    # TODO - phase 4 - this can be automated with gh
    # https://cli.github.com/manual/gh_release_create
    # TODO - can we pull the list of changes from Changelog.md and automatically add those to the release (so we do not type the same things twice)
    # TODO - if/when we automate this, make sure to open the releases page so the user can review it.
```

Note that we don't just copy all the changes from the changelog, usually. We only do "highlights"